### DISCUSSION TITLE
Native Ads

### DISCUSSION THREAD

**Discussion by runrundeals** [admob-plus/admob-plus#491] [Ideas]
Native Ads Will it be added to your plugin?
Our Application Infrastructure ionic
We would be very grateful if you could inform us about this.

**Comment by sevkonline** [comment]
i need the same thing. Waiting for native ads to be added for IONIC Cordova.

**Reply by EYALIN** [reply]
@ratson if we will donate certain amount, would you be able to add the native ad?
